evm/logic/system.py

```diff
@@ -261,7 +260,8 @@ def paygas(computation):
             PAYGAS_gasprice = computation.get_PAYGAS_gas_price()
             if PAYGAS_gasprice is None:
                 PAYGAS_gasprice = 0
-            fee_to_be_charged = PAYGAS_gasprice * computation.msg.transaction_gas_limit
+            fee_to_be_charged = PAYGAS_gasprice * \
```

Comment:
We've currently got zero `\` based multi-line statements (that I know of).  Prefered code style for this is.

```python
fee_to_be_charged = (
    PAYGAS_gasprice * computation.transaction_context.transaction_gas_limit
)
# or
gas_limit = computation.transaction_context.transaction_gas_limit
fee_to_be_charged = PAYGAS_gasprice * gas_limit
```

Reply:
Thanks. I actually looked for this in the tactical manual and couldn't find it :)